src/Domain/LeanCode.CQRS.Execution/DevResponseLoggerElement.cs

```diff
@@ -0,0 +1,34 @@
+
+using LeanCode.Contracts;
+using LeanCode.Contracts.Security;
+using LeanCode.Pipelines;
+using Microsoft.Extensions.Hosting;
+
+namespace LeanCode.CQRS.Execution
+{
+    public class DevResponseLoggerElement<AppContext, TObj, TRes> : IPipelineElement<AppContext, TObj, TRes>
+        where AppContext : notnull, IPipelineContext
+    {
+        public readonly Serilog.ILogger logger = Serilog.Log.ForContext<DevResponseLoggerElement<AppContext, TObj, TRes>>();
+        public readonly IHostEnvironment environment;
+
+        DevResponseLoggerElement(IHostEnvironment env)
+        {
+            logger = Serilog.Log.ForContext<DevResponseLoggerElement<AppContext, TObj, TRes>>();
```

Comment:
```suggestion
```

You already initializing it in a field initializer.

You should have a separate ctor, that takes both `IHostEnvironment` and `ILogger` for tests only - but that one should be separate from the one that should be used most of the time.